imports/ui/components/Utilities/Alerts/AlertMessage.vue

```diff
@@ -36,6 +36,18 @@
 <script lang="ts">
 import { defineComponent, getCurrentInstance } from 'vue';
 
+export interface AlertMessageType {
+  showAlertSimple(color: string, title:  string): void;
+  showAlertFull(icon: string,
+                color: string,
+                title: string,
+                mode: string,
+                timeout: number,
+                location: string,
+                text?: string): void;
+  closeAlert(): void;
+}
```

Comment:
supposedly, this should not be necessary cause the component is using defineComponent which provides types to the exported component